modules/rtgtools/vcfeval/main.nf

```diff
@@ -0,0 +1,55 @@
+process RTGTOOLS_VCFEVAL {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::rtg-tools=3.12.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/rtg-tools:3.12.1--hdfd78af_0':
+        'quay.io/biocontainers/rtg-tools:3.12.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(truth_vcf), path(truth_vcf_tbi), path(query_vcf), path(query_vcf_tbi), path(bed)
+    path(sdf)
+
+    output:
+    tuple val(meta), path("*.txt"), emit: results
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def regions = bed ? "--bed-regions=$bed" : ""
+    def truth_index = truth_vcf_tbi ? "" : "rtg index $truth_vcf"
+    def query_index = query_vcf_tbi ? "" : "rtg index $query_vcf"
+
+    sdf_basename = sdf.getBaseName().replace(".tar","")
+    tar_decomp = ""
+    if((sdf =~ /.tar.gz\b/).find() == true) {
```

Comment:
Not sure this belongs here.
@nf-core/core is this allowed?

Reply:
It just checks if the SDF is tarzipped or not. Why would this not be allowed?

Reply:
The check is allowed, but I'm not sure if the untarring it allowed inside the same module.
Generally, you untar in a separate process and use the output as input for your new module

Reply:
e.g. https://github.com/nf-core/modules/blob/9ea7f725efb8534b9c0e91484f68e5a6dcda3e89/tests/modules/bclconvert/main.nf#L20

Reply:
Oooh, okay! Yeah that may be a better way to handle it. Thanks!